### DISCUSSION TITLE
Error while i try to destroy buttons in frame

### DISCUSSION THREAD

**Discussion by PeterLinuxOS** [TomSchimansky/CustomTkinter#2850] [Q&A]
I getting this error message when I try to delete buttons from the frame 
Error message
![image](https://user-images.githubusercontent.com/69091177/206280242-1f942f7c-9ebd-4a74-aaa7-3748b4871785.png)
Delete Button/s
![image](https://user-images.githubusercontent.com/69091177/206280945-7e5efba0-07b5-4b8b-a06c-2bc6323c2402.png)
Insert button
![image](https://user-images.githubusercontent.com/69091177/206281095-aec4cffb-6022-4911-9c40-b24bedb9a4fa.png)

**Comment by clxakz** [comment]
The variable button is not assigned to anything.
The code should be

`button = customtkinter.CTkButton(commend_frame_userlist, text=str(idb["steamID64]), fg_color="transparent", command=select_profile).pack(pady=2)`

**Comment by PeterLinuxOS** [comment]
> The variable button is not assigned to anything. The code should be
> 
> `button = customtkinter.CTkButton(commend_frame_userlist, text=str(idb["steamID64]), fg_color="transparent", command=select_profile).pack(pady=2)`

It's a local variable that is made by for loop and btw u cant use `.pack(pady=2)` when you define the button as the variable.
